makefile

```diff
@@ -79,10 +79,13 @@ fix-path:
 	@$(BIN_DIR)/fix-path.sh $(MAIN_DIR)
 	@$(BIN_DIR)/fix-path.sh $(TEST_DIR)
 
+# raml-for-jax-rs uses a jersey/jaxrs version prior to the package rename from javax to jakarta.
+# This is why we find/replace "javax.ws" in the generated code with "jakarta.ws"
 gen-jaxrs: api.raml merge-raml
 	@$(BIN_DIR)/generate-jaxrs.sh $(APP_PACKAGE)
 	@$(BIN_DIR)/generate-jaxrs-streams.sh $(GEN_PACKAGE)
 	@$(BIN_DIR)/generate-jaxrs-postgen-mods.sh $(GEN_PACKAGE)
+	@grep -Rl javax src | xargs -I{} sed -i 's/javax.ws/jakarta.ws/g' {}
```

Comment:
I'm not sure if there's a path to having the `raml-for-jax-rs` build script generate with the new package namespace.

Reply:
The grep/sed script you added could be moved to a shell script in the tools repo.